distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -28,10 +26,19 @@ polyglot java import org.enso.base.Array_Utils
 
          Vector.new my_vec.length (ix -> my_vec.at ix)
 new : Number -> (Number -> Any) -> Vector Any
-new length constructor =
-    builder = (0.up_to length).fold (new_builder length) builder-> ix->
-        builder.append (constructor ix)
-    builder.to_vector
+new length constructor = @Builtin_Method "Vector.new_builtin"
+
+## ADVANCED
+
+   Converts an array into a vector.
```

Comment:
Probably worth noting that it copies the contents.

Reply:
+1